RELEASE.md

```diff
@@ -66,10 +81,7 @@ This release continues from the 0.5.0 release of oqs-provider and is fully teste
 
 **Full Changelog**: https://github.com/open-quantum-safe/oqs-provider/compare/0.5.0...0.5.1
 
-Previous Release Notes
-======================
-
-This is version 0.5.0 of oqs-provider.
+# This is version 0.5.0 of oqs-provider.
```

Comment:
Spurious line?

Reply:
Nope -- just a previous release('s) heading. But the first line was wrong. Thanks for checking back!